src/bundler/bundle_v2.zig

```diff
@@ -2279,6 +2279,7 @@ pub const ParseTask = struct {
     known_target: ?options.Target = null,
     module_type: options.ModuleType = .unknown,
     ctx: *BundleV2,
+    package_version: ?string = null,
```

Comment:
this is 32 bytes, 16 bytes for string and due to alignment, an extra 16 bytes

we can do a regular string and just make it empty by default instead of null